src/main/java/org/ossgang/commons/monads/Maybe.java

```diff
@@ -105,31 +116,64 @@ public boolean hasException() {
         return exception != null;
     }
 
+    /**
+     * Returns true, if this Maybe is in a successful state, i.e. it does not contain an exception and thus has a valid
+     * value. For normal Maybes, the value will always be non-null. The only exception to this is a Maybe<Void> which in
+     * successful state has a value of {@code null}. This is equivalent to {@code !hasException()}
```

Comment:
"This is equivalent to {@code !hasException()}" - no, it is not for Maybe<Void>. I would just remove that mentioning here.

Reply:
Indeed ... was a nasty moving/copying error ... I changed.